src/components/Alert/styles/StyledDiv.js

```diff
@@ -1,5 +1,5 @@
 import styled, { css } from 'styled-components'
-import BigText from '../../BigText'
+import { BigText } from '../../Typography'
```

Comment:
It should be a `Typographies`

Reply:
Nevermind : 
https://en.wiktionary.org/wiki/typography
https://www.wordhippo.com/what-is/the-plural-of/typography.html